Checkr/src/components/atoms/Typography/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Typography, { TypographyProps } from '@mui/material/Typography';
+import { useTheme } from '@mui/material/styles';
+import { Theme } from '@mui/material/styles';
+
+interface AtomTypographyProps extends TypographyProps {
+  text?: string;
+}
+
+const AtomTypography: React.FC<AtomTypographyProps> = (props) => {
```

Comment:
Typography would be fine. rename the component

Reply:
ok